docs/api/alerts/update.asciidoc

```diff
@@ -0,0 +1,108 @@
+[[alerts-api-update]]
+=== Update alert API
+++++
+<titleabbrev>Update alert</titleabbrev>
+++++
+
+Update the attributes for existing {kib} alerts.
+
+[[alerts-api-update-request]]
+==== Request
+
+`PUT <kibana host>:<port>/api/alerts/alert/<id>`
+
+[[alerts-api-update-path-params]]
+==== Path parameters
+
+`id`::
+  (Required, string) The alert ID to update.
+
+[[alerts-api-update-request-body]]
+==== Request body
+
+`name`::
+  (Required, string) A name to reference and search in the future.
+
+`tags`::
+  (Optional, string array) A list of keywords to reference and search in the future.
+
+`schedule`::
+  (Required, object) The schedule specifying when this alert should be run, using one of the available schedule formats specified under _Schedule Formats_ below.
+
+`throttle`::
+  (Optional, string) A Duration specifying how often this alert should fire the same actions. This will prevent the alert from sending out the same notification over and over. For example, if an alert with a `schedule` of 1 minute stays in a triggered state for 90 minutes, setting a `throttle` of `10m` or `1h` will prevent it from sending 90 notifications over this period.
```

Comment:
```suggestion
  (Optional, string) How often this alert should fire the same actions. This will prevent the alert from sending out the same notification over and over. For example, if an alert with a `schedule` of 1 minute stays in a triggered state for 90 minutes, setting a `throttle` of `10m` or `1h` will prevent it from sending 90 notifications during this period.
```